mobile/lib/features/wallet/share_invoice_screen.dart

```diff
@@ -65,13 +66,14 @@ class _ShareInvoiceScreenState extends State<ShareInvoiceScreen> {
                     style: TextStyle(fontSize: 18.0, fontWeight: FontWeight.bold),
                   ),
                 ),
-                Expanded(
-                  child: Center(
-                    child: QrImageView(
-                      data: widget.invoice.rawInvoice,
-                      version: QrVersions.auto,
-                      padding: const EdgeInsets.all(qrPadding),
-                    ),
+                Container(
+                  height: qrWidth,
```

Comment:
This line implies that `qrWidth` is the width of the QR code: https://github.com/get10101/10101/blob/b91fa44a02625b4d9efd5c61323f4f269cd53b3d/mobile/lib/features/wallet/share_invoice_screen.dart#L82

But that actually isn't true :sweat_smile:  So forcing it into these dimensions keeps it quite small

Reply:
On main:
![image](https://github.com/get10101/10101/assets/6688948/94327c37-1dc3-4a60-82fb-1aba8f4d8dfc)

On this branch: 

![image](https://github.com/get10101/10101/assets/6688948/f820a54e-bcc6-4018-bd62-aabe9bb9844b)

The "Amount: ..." text has got nothing to do with the QR code size on `main`, but it works as per the comment on this branch. This makes me think the original code is buggy and this PR just "fixes" that bug.

The reason this is needed in the first place is because the QrImageView causes errors in flutter if it isn't assigned a height and it is a child of an `IntrinsicHeight` widget (as used here to prevent specifying the whole viewport height manually)

Reply:
I wonder how that looks on the phone. I guess the new version wastes too much screen real state at the moment.

Could we just skip changing this page for now? I don't think one can open the keyboard on this screen anyway.

Reply:
> I wonder how that looks on the phone. I guess the new version wastes too much screen real state at the moment.

Well, the phone usually won't be as big as on desktop anyway 😅 so I doubt it makes much of a difference

> Could we just skip changing this page for now? I don't think one can open the keyboard on this screen anyway.

We can, yea